bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -395,6 +397,11 @@ pub mod pallet {
 					),
 					Error::<T>::InvalidSyncCommitteeMerkleProof
 				);
+			} else {
+				ensure!(
+					update_finalized_period == store_period,
+					<Error<T>>::SyncCommitteeUpdateRequired
```

Comment:
Lets please standardize on this style, which we mostly use elsewhere

```suggestion
					Error::<T>::SyncCommitteeUpdateRequired
```